bolt-design-system/bolt
# Task/update guides

## Jira

n/a

## Summary

Edit the guides to setting up Bolt locally.

## Details

 - Changed curly quotes to straight quotes to avoid copy-paste issues in terminal
 - Removed deprecated 'homebrew-php' package, which has been migrated into 'homebrew-core'
 - Added comment after PHP installation which ensures terminal is referencing the right version of PHP

## How to test
 - Go to https://boltdesignsystem-erwjmykywp.now.sh/docs/guides/preparing-your-environment.html
 - See straight quotes added to first 2 code snippets
 - On step 4, see new comment after PHP installation, see 'homebrew-php' has been removed
